lib/raven/configuration.rb

```diff
@@ -375,7 +375,10 @@ def detect_release_from_capistrano
     end
 
     def detect_release_from_git
-      `git rev-parse --short HEAD`.strip if File.directory?(".git") rescue nil
+      release = `git rev-parse --short HEAD 2>/dev/null`.strip if File.directory?('.git')
+      return nil if release == ''
+    rescue
```

Comment:
New RuboCop rules would say "Ah, you ought to catch specific errors, such as `StandardError`, in this case".

Reply:
Oh, and `C: Class has too many lines.` just happened.